linkcheck/url.py

```diff
@@ -28,6 +28,7 @@
     import urlparse
 
 import requests
+from builtins import str as text
```

Comment:
(I would even suggest `text_type`, but that might be just that I'm so used to [`six`](https://pypi.org/project/six/))